CONTRIBUTING.md

```diff
@@ -0,0 +1,40 @@
+# Contributing
+
+Thanks for stopping by! This document should cover most topics surrounding contributing to this repo.
+
+* [How to contribute](#how-to-contribute)
+  * [Reporting issues](#reporting-issues)
+  * [Fixing issues](#fixing-issues)
+  * [Adding features](#adding-features)
+
+## Reporting issues
+Are you having trouble getting started? Please [contact us directly](mailto:support@bugsnag.com?subject=%5BGitHub%5D%20bugsnag-js-performance%20-%20having%20trouble%20getting%20started%20with%20BugSnag) for assistance with integrating BugSnag into your application.
+If you have spotted a problem with this module, feel free to open a [new issue](https://github.com/bugsnag/bugsnag-js-performance/issues/new?template=Bug_report.md). Here are a few things to check before doing so:
+
+* Are you using the latest version of BugSnag? If not, does updating to the latest version fix your issue?
+* Has somebody else [already reported](https://github.com/bugsnag/bugsnag-js-performance/issues?utf8=%E2%9C%93&q=is%3Aissue%20is%3Aopen) your issue? Feel free to add additional context to or check-in on an existing issue that matches your own.
+* Is your issue caused by this module? Only things related to `@bugsnag/browser-performance`, `@bugsnag/react-native-performance`, and performance related plugins (such as `@bugsnag/plugin-react-navigation-performance`) should be reported here. For anything else, please [contact us directly](mailto:support@bugsnag.com) and we'd be happy to help you out.
+
+### Fixing issues
+
+If you've identified a fix to a new or existing issue, we welcome contributions!
+Here are some helpful suggestions on contributing that help us merge your PR quickly and smoothly:
+
+* [Fork](https://help.github.com/articles/fork-a-repo) the
+  [library on GitHub](https://github.com/bugsnag/bugsnag-js-performance)
+* Build and test your changes. We have automated tests for many scenarios but its also helpful to use `npm pack` to build the module locally and install it in a real app.
+* Commit and push until you are happy with your contribution
+* [Make a pull request](https://help.github.com/articles/using-pull-requests)
+* Ensure the automated checks pass (and if it fails, please try to address the cause)
```

Comment:
If I understand correctly, our pipeline doesn't actually build PRs raised by external users. Should we drop the line about automated tests?